npm_nodejs/std_node.md

```diff
@@ -52,7 +52,7 @@ If you want documentation for any of the modules, you can simply type `deno doc`
 and the URL of the module in your terminal:
 
 ```
-> deno doc https://deno.land/std/assert.ts
+> deno doc https://deno.land/std/node/assert.ts
```

Comment:
```suggestion
> deno doc https://deno.land/std/fs/move.ts
```
This is shorter and more welcoming on the terminal.
<img width="549" alt="terminal" src="https://user-images.githubusercontent.com/29819102/129852917-32447157-b3e7-4562-a093-2b76876f42cf.png">